.github/workflows/pr-analysis.yml

```diff
@@ -1,14 +1,14 @@
 name: PR Analysis
 on:
   pull_request:
-    types: [opened, synchronize, labeled, unlabeled]
+    types: [labeled, unlabeled]
```

Comment:
I think this can help to avoid multiple unnecessary runs

Reply:
That technically means that if something is opened with the do-not-merge label present then it can still go in, doesn't it?

Reply:
You're right, I didn't consider that case  :)